Application/Dependency Injection/DependencyInjection.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Reflection;
+using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Application.Dependency_Injection
+{
+    public static class DependencyInjection
+    {
+        public static void AddApplication(this IServiceCollection services)
+        {
+            services.AddMediatR(Assembly.GetExecutingAssembly());
```

Comment:
I think this class is superfluous. Better just addMediatR in startup

Reply:
Resolved